### ISSUE TITLE
[App Performance] Low Performance Issues

### ISSUE THREAD

**Issue by hmaltr** [Dar9586/NClientV2#142]
When searching a manga or loading a page (main page) the app stops for 5-6 secs then continues as normal, before it was progressive and more responsive. Can you optimize it a little bit? Thanks.

**Comment by hmaltr** [comment]
Samsung A50 running Android 10

**Comment by Dar9586** [comment]
I don't know if I fixed this by mistake, but testing on:
- Emulator API 29 (Android 10)
- Emulator API 16 (Android 4.1)
- Physical device API 28 (Android 9)
It didn't give me any performance issues with version 2.3.5, if it continues being slow maybe it is the connection with the nhentai/nhent.ai site

**Comment by hmaltr** [comment]
Yes it was the connection but the app freezes and even android ANR appears.
Can you make it progressive? Instead of waiting to load everything?

**Comment by hmaltr** [comment]
@Dar9586 what did you change 2.3.4? The app lags in the newer versions.
